### ISSUE TITLE
Accomodate multiple kitspace projects in one repo

### ISSUE THREAD

**Issue by patcon** [kitspace/kitspace#149]
For those who've either go a few in one, or are supporting multiple versions (just different folders, in this case), and want to keep project in one authoritative git history.

Our project has multiple versions that we might like make available, and also a separate badge programmer:
https://github.com/HumanDynamics/openbadge/tree/master/eagle/nRF_badge

Thanks for the consideration! Happy to submit a PR when the time comes, if this is something you think might be supportable :)

**Comment by kasbah** [comment]
The way I would handle multiple versions in my own projects is through git tags. I am planning some UI for this and navigating the git history in general in the future.

Multiple projects per repo would still be useful though. I think in the kitspace.yaml it could be handled by allowing something like:

```yaml
multi:
    project_1:
         gerbers: project_1/gerbers
    project_2:
        gerbers: project_2/gerbers
```

**Comment by Jan--Henrik** [comment]
I also need this feature to add my projects to Kitspace.org, accordingly +1 :)

**Comment by kasbah** [comment]
This was fixed with #180 !
